script/Ciel/Day 13/13_01_01_CIEL13_1.txt

```diff
@@ -740,22 +794,23 @@ The situation is already beyond repair.
 [sha:f2eb9a30e08a5a49c221254f78d8c86663295261]{
 -- Page 39, Offset 33213.
 -- あの夢のおかげだ。
-It's because of that dream.
+That dream made it clear to me.
 }
 [sha:2890f504add2cd990cc81ab08636f3346363e599]{
 -- Page 39, Offset 33214.
 -- 前の<転生先|ヤツ>の気持ちを経験として理解する。
-I understand after experiencing the feelings of the previous incarnate.
+Now that I've experienced the feelings of the previous vessel, I understand.
 }
 [sha:d22ac954e552b7d1fc59e0dfa5587168411277a9]{
 -- Page 39, Offset 33215.
 -- 俺が死んだところで死ぬのは俺のイシだけで、残ったロアのイシがこの体を完全に操るだろう。
-Even if I were to die, the only thing that would die is my own will, whereas Roa will end up completely controlling my body.
+If I killed myself, the only thing to die would be my own will.
+ Roa would remain, now in full control of the body I ceded to him.
```

Comment:
The first sentence had a bit of a weird structure.
I think you want the intentional contrast of the two wills. My will <> Roa's will
>If I killed myself, my will would be the only thing that died.
>Roa's will would remain...